.github/workflows/azure-kubernetes-service-helm.yml

```diff
@@ -1,19 +1,19 @@
 name: Build and deploy matrix webserver app to AKS with Helm
 
 on:
-  push:
+  # push:
+  pull_request:
     branches:
       - master
```

Comment:
Now that we know this works we should move it back to just deploy on merge to master.

Reply:
Done -- see commit "reverts trigger back to deploying when merge to master"